config/routes.rb

```diff
@@ -128,6 +128,7 @@
     end
 
     namespace :v3, constraints: ->(_request) { FeatureFlag.active?(:api_v3) } do
+      resources :statements, only: %i[index], controller: "finance/statements"
```

Comment:
I think this might work and is slightly shorter, but that's really just stylistic so feel free to ignore
```suggestion
      resources :statements, only: %i[index], module: "finance"
```